packages/playwright/src/runner/loadUtils.ts

```diff
@@ -28,10 +29,12 @@ import type { TestRun } from './tasks';
 import { requireOrImport } from '../transform/transform';
 import { applyRepeatEachIndex, bindFileSuiteToProject, filterByFocusedLine, filterByTestIds, filterOnly, filterTestsRemoveEmptySuites } from '../common/suiteUtils';
 import { createTestGroups, filterForShard, type TestGroup } from './testGroups';
-import { dependenciesForTestFile } from '../transform/compilationCache';
+import { affectedTestFiles, dependenciesForTestFile } from '../transform/compilationCache';
 import { sourceMapSupport } from '../utilsBundle';
 import type { RawSourceMap } from 'source-map';
 
+const toPosixPath = (s: string) => s.replaceAll(path.sep, path.posix.sep);
```

Comment:
Might as well use the utility one: https://github.com/microsoft/playwright/blob/main/packages/playwright-core/src/utils/fileUtils.ts#L59

Reply:
done in https://github.com/microsoft/playwright/pull/31727/commits/b6ef01324fa3dd0dc6bf70d348bb38d409ce7c55